docs/docs/native-docker.md

```diff
@@ -280,6 +280,58 @@ the future, as Mesos may not always interact with Docker via the CLI.
 }
 ```
 
+#### Docker container support without Docker Engine
+
+Starting with version 1.2.0, Marathon supports docker container images without having the Docker Containerizer
+depend on a Docker Engine. Instead the Mesos containerizer with native AppC support added in Apache Mesos version 1.0
+(released July 2016) directly uses native OS features to configure and start Docker containers and to provide isolation.
+This setup is selected by specifying a JSON combination that used to provoke an error message:
```

Comment:
I am not sure whether extending the v2 container spec is a nice way to go. Several thoughts here:
- we will start working on v3 _very soon_ and are about to deprecate v2
- the container spec is already overloaded and this difficult to parse and validate
- adding a new json node with a specific set of available properties would not ease validation and prevent extra effort in the MarathonUI needed to cope with new combinations that were invalid before.